src/js/popup.js

```diff
@@ -212,6 +212,11 @@ const Logic = {
     return false;
   },
 
+  async noOfTabs() {
```

Comment:
nitpick: can the function name be `numOfTabs`?

Reply:
sure :)

Reply:
> nitpick: can the function name be `numOfTabs`?

Sure :)